pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -499,6 +499,7 @@ type ConsulSDConfig struct {
 	// +optional
 	Services []string `json:"services,omitempty"`
 	// An optional list of tags used to filter nodes for a given service. Services must contain all tags in the list.
+	// Deprecated: As of Consul 1.14, use `filter` with the `ServiceTags` selector instead.
```

Comment:
nit: Its not clear to me about `ServiceTag` selector. If its consul specific should we keep minimal info?

Reply:
ref: https://developer.hashicorp.com/consul/api-docs/catalog#query-parameters-4
It is necessary to remind users how to rewrite `Tags` and `NodeMeta` into `Filter`.